aws-msk/alerts/baseline-alert.yml

```diff
@@ -0,0 +1,65 @@
+---
+
+# Name of the alert
+name: Name of this alert condition
```

Comment:
This appears to be a copy/paste of the template quickstart alerts, did you want to include alerts in this quickstart or should these just be removed?

Reply:
I had only the dashboard to submit. You may remove the alert config